### PR TITLE
data-generator-retail - rename commands to orders

### PR DESCRIPTION
**Pull request by djhi**

### CODE REVIEW

**Comment by fzaninotto** [comment]
there are still instances of command_id in the demo, as well as instances of the 'commands' keyword

**Review by fzaninotto — COMMENTED** [review]
When I test the demo in this branch, any request for the orders returns an empty response.

**Review by fzaninotto — APPROVED** [review]

File: examples/demo/src/dataProvider/graphql.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ const getGqlResource = (resource: string) => {
         case 'categories':
             return 'Category';
 
-        case 'commands':
+        case 'orders':
             return 'Command';
```
[the remark this answers is not in the document]
Comment (djhi) [reply examples/demo/src/dataProvider/graphql.ts:19]: Indeed :ok_hand:

**Review by adguernier — APPROVED** [review]

File: examples/demo/src/dataProvider/graphql.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (djhi) [reply examples/demo/src/dataProvider/graphql.ts]: Fixed

**Pull request merged**